google/cloud/ids/README.md

```diff
@@ -3,7 +3,12 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Cloud IDS API][cloud-service-docs], a service to Cloud IDS (Cloud Intrusion Detection System) detects malware, spyware, command-and-control attacks, and other network-based threats. Its security efficacy is industry leading, built with Palo Alto Networks technologies. When you use this product, your organization name and consumption levels will be shared with Palo Alto Networks.
+[Cloud IDS (Intrusion Detection System) API][cloud-service-docs], a
```

Comment:
I think acronyms are typically: "Intrusion Detection System (IDS)"... then just "IDS" thereafter.

Reply:
We were just copying the text from the documentation summary in the service yaml file.

But that said, I agree with you and I'll change it when I regenerate this code after #7926 lands.  Thanks.

Reply:
Oops ... no ... I got confused between "ids" and "iot".  :-)

I'll correct this here/now.

Reply:
Done.